### PR TITLE
Era 14/budget alarms

### PR DESCRIPTION
**Pull request by BerradaYassine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,26 @@ In order to keep the repository clean. Untagged images are deemed as non-essenti
...
+
+#### **Budget Alarms**
+
+Following this ideology of rightsizing and FinOps optimization, we provided in this section 8 Alarms monitoring the 4 pillars of this package which are backend, frontend, rds, vpc. Once the pillars exceed the threshold set, you receive a notification through different emails of your choosing, in order for the company to look into the spending culture inside the aws accounts.
+
+these are the threshold set by default for the actual spending, again customizable at will.
```
Comment (flyck) [inline README.md:22]: Sentence needs to start captialized, please end it with a colon

File: README.md [hunk 2]
```diff
@@ -9,3 +9,26 @@ In order to keep the repository clean. Untagged images are deemed as non-essenti
...
+
+#### **Budget Alarms**
+
+Following this ideology of rightsizing and FinOps optimization, we provided in this section 8 Alarms monitoring the 4 pillars of this package which are backend, frontend, rds, vpc. Once the pillars exceed the threshold set, you receive a notification through different emails of your choosing, in order for the company to look into the spending culture inside the aws accounts.
```
Comment (flyck) [inline README.md:20]: Is FinOps an ideology or more a practice?

File: README.md [hunk 3]
```diff
@@ -9,3 +9,26 @@ In order to keep the repository clean. Untagged images are deemed as non-essenti
...
+
+these are the threshold set by default for the actual spending, again customizable at will. 
+
+- Rds : 50 USD
+- Frontend : 80 USD
+- Backend : 100 USD 
+- Vpc : 120 USD
+
+And as of the 4 other alarms are for the Forecasted which send another email once your spending is expected to exceed the threshold priorly showcased set by 3 times you also get notified in advance, in order to take quick action into it. refer to the forecasted threshold below :
```
Comment (flyck) [inline README.md:29]: please remove the space before the colon. "Refer" needs to be capitalized

File: README.md [hunk 4]
```diff
@@ -9,3 +9,26 @@ In order to keep the repository clean. Untagged images are deemed as non-essenti
...
+- Frontend : 80 USD
+- Backend : 100 USD 
+- Vpc : 120 USD
+
+And as of the 4 other alarms are for the Forecasted which send another email once your spending is expected to exceed the threshold priorly showcased set by 3 times you also get notified in advance, in order to take quick action into it. refer to the forecasted threshold below : 
+
+- Rds forecasted : 150 USD 
+- Frontend forecasted : 240 USD 
+- Backend forecasted : 300 USD
+- Vpc forecasted : 360 USD
```
Comment (flyck) [inline README.md:34]: A link is missing. Where do people change these values?

File: dev/backend/variables.tf [hunk 5]
```diff
@@ -16,3 +16,15 @@ variable "oauth_token" {
 variable "region" {
   type = string
 }
+
+variable "email_adresses" {
+    type = list(string)
+    description = "list of email addresses to be subscribed to get the notification of the budget alarm"
+    default = ["exemple-email-1@gmail.com","exemple-email-2@gmail.com"]
```
Comment (flyck) [inline dev/backend/variables.tf:23]: Let's remove the default option here, we dont want to be sending spam email to random addresses.

Comment (BerradaYassine) [reply dev/backend/variables.tf:23]: yes it was just for testing

**Comment by flyck** [comment]
LGTM!

**Pull request merged**